src/main/java/seedu/address/model/person/MedCon.java

```diff
@@ -46,4 +46,9 @@ public boolean equals(Object other) {
     public int hashCode() {
         return value.hashCode();
     }
+
+    @Override
+    public int compareTo(MedCon other) {
+        return this.value.compareTo(other.value);
+    }
```

Comment:
maybe just add in the header javadoc that this class's natural ordering is alphabetical order of value